pom.xml

```diff
@@ -102,6 +102,42 @@
             <artifactId>jopt-simple</artifactId>
             <version>5.0.3</version>
         </dependency>
+
```

Comment:
remove `commons-exec` dependency, it is unused now

Reply:
Done.